packages/in_app_purchase_tizen/tizen/src/billing_manager.cc

```diff
@@ -0,0 +1,264 @@
+#include "billing_manager.h"
+
+#include <flutter/method_channel.h>
+#include <flutter/standard_method_codec.h>
+
+#include <cassert>
+#include <iomanip>
+#include <sstream>
+#include <string>
+#include <vector>
+
+#include "log.h"
+
+static server_type ConvertServerType(const char *server_type_string) {
+  if (strcasecmp("DEV", server_type_string) == 0) {
+    return SERVERTYPE_DEV;
+  } else if (strcasecmp("OPERATE", server_type_string) == 0) {
+    return SERVERTYPE_OPERATE;
+  } else if (strcasecmp("WORKING", server_type_string) == 0) {
+    return SERVERTYPE_WORKING;
+  } else if (strcasecmp("DUMMY", server_type_string) == 0) {
+    return SERVERTYPE_DUMMY;
+  } else
+    return SERVERTYPE_NONE;
+}
+
+void BillingManager::Init() {
+  billing_api_handle_ = OpenBillingApi();
+  if (billing_api_handle_ == nullptr) {
+    LOG_ERROR("Fail to open billing api");
```

Comment:
If fail to open billing api, we need return.

Reply:
fixed